### PR TITLE
az-version: add page

### PR DESCRIPTION
**Pull request by derNiklaas**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.40.0

Part of [#6141](https://github.com/tldr-pages/tldr/issues/6141)

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/az-version.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# az version
+
+> Shows the current version of Azure CLI modules and extensions.
+> Part of `az`, the command-line client for Microsoft Azure.
+> More information: <https://learn.microsoft.com/en-us/cli/azure/reference-index?view=azure-cli-latest#az-version>.
```
Comment (navarroaxel) [inline pages/common/az-version.md:5]: We don't include i18n in MSFT URLs
```suggestion
> More information: <https://learn.microsoft.com/cli/azure/reference-index?view=azure-cli-latest#az-version>.
```

File: pages/common/az-version.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+- Show the current version of Azure CLI modules and extensions:
+
+`az version`
+
+- Show the current version of Azure CLI modules and extensions in a given format (default JSON):
```
Comment (navarroaxel) [inline pages/common/az-version.md:11]: I think the default is useless here
```suggestion
- Show the current version of Azure CLI modules and extensions in a given format:
```

File: pages/common/az-version.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# az version
+
+> Shows the current version of Azure CLI modules and extensions.
+> Part of `az`, the command-line client for Microsoft Azure.
+> More information: <https://learn.microsoft.com/en-us/cli/azure/reference-index?view=azure-cli-latest#az-version>.
+
+- Show the current version of Azure CLI modules and extensions:
```
Comment (navarroaxel) [inline pages/common/az-version.md:7]:
```suggestion
- Show the current version of Azure CLI modules and extensions in JSON format:
```

**Review by navarroaxel — COMMENTED** [review]
Thanks @derNiklaas. I left some comments

**Review by CleanMachine1 — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**